### ISSUE TITLE
[469c-RC2] UnrealEd Crash (using Wine/Gentoo Linux)

### ISSUE THREAD

**Issue by RoninMastaFX** [OldUnreal/UnrealTournamentPatches#946]
Hey there! :)

I installed the Linux (32 and 64-bit) & Windows versions of 469c RC2, and all works well with the game on the Linux 32-bit & 64-bit versions alongside the Windows version (via Wine 6.0.2, using Gentoo LInux). However, when I launch the UnrealEd, I get this error:
![Screenshot_20220722_161258](https://user-images.githubusercontent.com/3757396/180552956-1c6df601-6307-449e-bc8a-db96151d4f32.png)

I let UT recreate the UnrealTournament.ini file alongside the UnrealEd.ini file, but to no avail (aka same thing happened when launching the UnrealEd after). UT 469b's UnrealEd worked just fine with this version of Wine that I am using.

EDIT: Forgot to mention, if I click Ignore, it lets me into Recovery Mode, FWIW.

**Comment by SeriousBuggie** [comment]
Try set SoftDrv in UnrealEd.ini everywhere.
`Device=SoftDrv.SoftwareRenderDevice`
Instead of
`Device=OpenGLDrv.OpenGLRenderDevice`

**Comment by RoninMastaFX** [comment]
Thanks for your help Buggie, appreciate it bro. :)

I get a slightly different error message now, with SoftDrv replacing OpenGlDrv:
![Screenshot_20220722_164049](https://user-images.githubusercontent.com/3757396/180564487-64abab30-cead-434a-9282-4a5ba7f87914.png)

Also of note, as I said above, regenerating the UnrealEd does it partially. With a brand new UnrealTournament.ini and my trusty old UT INI, this version of UnrealEd still complains. I must be doing something stupid, unsure.

Thanks for the help!

**Comment by SeriousBuggie** [comment]
You replace all occurrences in UnrealEd.ini? Not just first one?

If so, try do same in UnrealTournament ini:
Set SoftDrv.SoftwareRenderDevice instead of OpenGLDrv.OpenGLRenderDevice everywhere except section names (names in square brackets).

**Comment by RoninMastaFX** [comment]
Buggie, you da man bro! That fixed it! How in the hell though, why would this be the problem... o_O

I just launched UT, it went into Software Rendering mode, then I changed back to OpenGL< worked fine ingame. I then launched the UnrealEd, she works good too!

I noted that this line here was likely the cause: 
RenderDevice=SoftDrv.SoftwareRenderDevice

I had that set as OpenGL before, just a heads up to all who are having this issue too.

I'm closing the issue with this, since the problem is solved. :)

Thanks again sir for your help!
